### DISCUSSION TITLE
Research into practitioner use?

### DISCUSSION THREAD

**Discussion by frankelavsky** [Myndex/SAPC-APCA#40] [Ideas]
Many research ventures that build tools (especially those based on existing empirical models) will validate their tool not just based on how well it performs against other models (which APCA does well) but also what practitioners think when using the tool.

I don't see validation studies for APCA and I think that if this is done as soon as possible, it could help improve APCA to be as simple and usable as possible. I really believe in this project, but there could be a chance to study and improve how it is used.

[The plain English version is really great](https://github.com/Myndex/SAPC-APCA#the-plain-english-steps-are)! As a starting point, perhaps bring in expert accessibility analysts and have them use WCAG 2.x contrast calculator (as a control/comparison) and then have them use APCA. This will be a great chance to gather thoughts about what could be improved/etc (like from a qualitative empirical perspective at the bare minimum). This is also a chance to put some research out on APCA (as a beta/prototype).

Since WCAG is primarily a document that gains its power from international political consensus, convincing political bodies that APCA is easily usable by practitioners is important. This study will go a long way in making the case that it should be in WCAG 3 and that WCAG 3 should be the new policy standard.

If you need research resources, perhaps there is an opportunity to connect and collaborate on this.

**Comment by Myndex** [comment]
This is the purpose of the public beta, and the public beta is what created all the fuss. LOL.

I have written so much on this, I am aware that things need to be "collated" but this project is underfunded to say the least and sorting out all that is low on the priorities list at the moment.

There is the draft in the WCAG_3 document, there are [THESE EARLY EXPLAINER DRAFTS](https://www.myndex.com/DRAFT/)

There is [THIS COLOR VISION DEFICIENCY SIMULATOR](https://www.myndex.com/CVD/)

Regarding "usability" APCA could be setup as simply as the current — in fact, Bridge-PCA is a WCAG_2 replacement that is 100% backwards compatible with WCAG_2 but using APCA technology.

At the SAPC site, you'll see the full font matrix, which is a technology demonstrator as part of the discussion of the multi-level WCAG_3 conformance model.

And anywhere in between. That is a matter of the standards implementation more than anything.

Please feel free to ask further questions here.

Thank you,

Andy

**Comment by Myndex** [comment]
To add:

There are a number of interfaces for APCA, and there will be more this week even. We've had a number of comments regarding how to use and so forth that drove the present design. They are closed issues in this tab, though some were moved to "Discussions"

I am going to move THIS thread to "Discussions" — this ISSUES tab is manly for specific bugs or specific actionable features or mods.
